python/src/helpers/s3.py

```diff
@@ -0,0 +1,46 @@
+import boto3
+import json
+from config import get_config
+import os
+
+import aws_xray_sdk as xray
+
+config = get_config()
+
+
+def get_cell_sets(experiment_id):
+    dir_path = os.path.join(config.LOCAL_DIR, f"{experiment_id}")
+
+    path_exists = os.path.exists(dir_path)
+
+    if not path_exists:
+        os.makedirs(dir_path)
+
+    with open(f"{dir_path}/cell_sets.json", "wb+") as f:
+        s3 = boto3.client("s3", **config.BOTO_RESOURCE_KWARGS)
+
+        print("downloading cellsets")
+
```

Comment:
Could you make these print out the timestamps as well, like in some of the other logs?